src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -387,6 +407,50 @@ class RecipeMarkdownGenerator : Runnable {
         }
     }
 
+    private fun addInfosFromManifests(recipeOrigins: Map<URI, RecipeOrigin>, cl: ClassLoader) {
+        val mfInfos = cl.getResources("META-INF/MANIFEST.MF").asSequence()
```

Comment:
Call me old fashioned, but I think being able to see non trivial types helps readability, especially with multiple chained operations.
```suggestion
        val mfInfos: Map<URI, Pair<License, String>> = cl.getResources("META-INF/MANIFEST.MF").asSequence()
```